lib/dealership.rb

```diff
@@ -0,0 +1,37 @@
+class Dealership
+  attr_reader :name, :address, :inventory
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    @inventory.length
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    if inventory_count > 0
+      true
+    else
+      false
+    end
+  end
+
+  def cars_by_make(make)
+    same_make = []
+    @inventory.each do |inventory|
+      if inventory.car.make == make
```

Comment:
This should read:
`if inventory.make == make` instead of `inventory.car.make`, because in this scope your block variable is `inventory`, which maybe to read better should just be `car`, since your inventory is an array of Cars. 

This method throws an error where Ruby is saying, "There is no "car" on `inventory`", because "inventory" is actually the car.